### PR TITLE
migrate lsp docs to manual and update for v2 registry completions

### PR DESCRIPTION
**Pull request by kitsonk**
~~Blocked by denoland/deno#13023~~

### CODE REVIEW

File: language_server/imports.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+`true`:
+
+- `https://deno.land/` - both the 3rd party `/x/` registry and the `/std/`
+  library registry are available.
+- `https://nest.land/` - a module registry for Deno on the blockweave.
+- `https://crux.land/` - a free open-source registry for permanently hosting
+  small scripts.
+
+```
+```
```
Comment (dsherret) [inline language_server/imports.md:342]: Remove?

File: language_server/imports.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+            "documentation": "https://api.example.com/docs/packages/${package}/${{version}}/paths/${path}",
+            "url": "https://api.example.com/packages/${package}/${{version}}/paths/${path}"
+          }
+        ]
+      }
+    ]
+  }
+  ```
+
+##### URL endpoints
```
Comment (dsherret) [inline language_server/imports.md:195]: I don't know if I've ever seen 5 `#` 😄

File: language_server/imports.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
+## Import completions and intelligent registries
```
Comment (dsherret) [inline language_server/imports.md:1]: Maybe we should use 1 `#` like the other files?

**Review by dsherret — APPROVED** [review]
LGTM

File: language_server/imports.md (hunk 3 above)
Comment (kitsonk) [reply language_server/imports.md:1]: Actually, most other sub pages use `##` instead of `#`.  It is only the top level "chapters" that use `#`.

**Pull request merged**